acs-core/src/main/java/com/arrow/acs/client/model/ProductModel.java

```diff
@@ -18,6 +21,7 @@
 	private String systemName;
```

Comment:
In general, it is desired to re-generate serialVersionUID if change class fields